proto/errorpb.proto

```diff
@@ -161,6 +161,13 @@ message MismatchPeerId {
     uint64 store_peer_id = 2;
 }
 
+message FollowerNotReady{
+    uint64 region_id = 1;
+    uint64 read_index = 2;
+    uint64 apply_index = 3;
+    uint64 peer_id = 4;
```

Comment:
Are they necessary for error handling? How are they used?

Reply:
yes, only used for debug, I will change it as reason.